src/control/server/instance_storage_test.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2020-2022 Intel Corporation.
+// (C) Copyright 2020-2023 Intel Corporation.
```

Comment:
Copyright seems to be the only change here, better to revert

Reply:
done